### PR TITLE
Stu2 ValueSet updates

### PR DESCRIPTION
**Pull request by Dtphelan1**
# Summary

Updates ValueSets(VS) to use STU2 equivalents. This PR also adds a markdown file and node-script, explaining and enabling the VS expansion pipeline for ease of future use. Since this expansion is scripted, a number of VS's beyond what we use today were generated; all VS based in the `elixhauser` IG were removed and all VS stemming from mCODE were left in, even if unused. 

Below, I highlight VS's we used , and what changes have been made.
- ValueSet-mcode-cancer-related-radiation-procedure-vs.json
  - Valueset was removed and replaced with several other radiotherapy valuesets
- ValueSet-mcode-cancer-related-surgical-procedure-vs.json
  - *Codes were removed*
- ValueSet-mcode-cancer-staging-system-vs.json
  - *Codes were added*
- ValueSet-mcode-primary-or-uncertain-behavior-cancer-disorder-vs.json
  - *Codes were updated*
  - *Codes were added*
- ValueSet-mcode-secondary-cancer-disorder-vs.json
  - *Codes were removed*
- ValueSet-mcode-tumor-marker-test-vs.json
  - *Codes were removed*
  - *Codes were added*

**Of note**: Several expansions failed, and instead of a VS with an `expansion` field we have the original `compose` field, intensionally defining the VS. Importantly, this happens to our cancer-related-surgical-procedure VS to the point of unusability. **We need to decide if we're okay with a janky Surgial-procedure extractor for now, or if we want to retain the old, more-usable VS to maintain Extractor functionality; while we discuss I've left the test broken**. 

Below is a full list of VS that fail expansion, with bolding applied to those VS we've had prior to this PR:
- *ValueSet-mcode-primary-or-uncertain-behavior-cancer-disorder-vs.json*
- ValueSet-mcode-radiotherapy-modality-vs.json
- ValueSet-mcode-radiotherapy-technique-vs.json
- ValueSet-mcode-radiotherapy-treatment-location-qualifier-vs.json
- ValueSet-mcode-radiotherapy-treatment-location-vs.json
- *ValueSet-mcode-secondary-cancer-disorder-vs.json*
- *ValueSet-mcode-tumor-marker-test-vs.json*

## New behavior

No new code behavior in the functional MEF; new behavior is restricted only to what codes are semantically tied to certain mCODE concepts. There is a new script for auto-expanding future VS – `vs-expansion-script.js`.

## Code changes

New vs-expansion-script should be reviewed.

# Testing guidance

- Ensure old tests still pass.
- Discuss and resolve open questions re: un-expandable VS.

### CODE REVIEW

**Comment by Dtphelan1** [comment]
The recent torrent of commits is the result of Julia, Julian, and me collaborating in the WS. With her careful feedback, I've successfully:
1. used the correct version of the spec; 
2. streamlined the automated-expansion process; 
3. manually expanded several VS, including 
  - secondary-cancer-disorder;
  - primary-or-uncertain-disorder; 
  - surgical-procedure
4. updated tests to resolve failures

We may still want to take a more sensible approach to 4, so I'm open to feedback.

**Review by jafeltra — COMMENTED** [review]
I had a few comments inline, but I had one other question in general:

- I think the Tumor Marker Test VS and Surgical Procedure VS were both ones we manually expanded, correct? Do you think there's a need to include the unexpanded filters in our ValueSets? I haven't actually looked into if there's a valid way to include both the expanded codes and additional filters in the FHIR resource, but I can if that's helpful in answering this question.

File: src/helpers/valueSets/vs-expansion-script.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+
+// NOTE: Update this to point to pre-expansion valuesets on your local machine
+const PREEXPANSIONPATH = path.resolve(__dirname, '../../../../fhir-mCODE-ig/fsh-generated/resources/valuesets');
```
[the remark this answers is not in the document]
Comment (jafeltra) [reply src/helpers/valueSets/vs-expansion-script.js:6]: I did some investigation with @Dtphelan1 and we realized that the absolute vs relative filepaths will be our downfall here. I don't think it's worth updating this script drastically to use commander and change how we parse args and build filepaths, so this is good as it is!

File: test/extractors/MCODERadiationProcedureExtractor.test.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 const { when } = require('jest-when');
-const rewire = require('rewire');
+// const rewire = require('rewire');
 const { MCODERadiationProcedureExtractor } = require('../../src/extractors/MCODERadiationProcedureExtractor.js');
 const exampleProcedureBundle = require('./fixtures/surgical-radiation-procedure-bundle.json');
 
-const radiationProcedureExtractorRewired = rewire('../../src/extractors/MCODERadiationProcedureExtractor.js');
-const getMCODERadiationProcedures = radiationProcedureExtractorRewired.__get__('getMCODERadiationProcedures');
+// const radiationProcedureExtractorRewired = rewire('../../src/extractors/MCODERadiationProcedureExtractor.js');
```
[the remark this answers is not in the document]
Comment (Dtphelan1) [reply test/extractors/MCODERadiationProcedureExtractor.test.js]: This has been resolved with option 3 – removing the extractor.

**Comment by jafeltra** [comment]
> * I think the Tumor Marker Test VS and Surgical Procedure VS were both ones we manually expanded, correct? Do you think there's a need to include the unexpanded filters in our ValueSets? I haven't actually looked into if there's a valid way to include both the expanded codes and additional filters in the FHIR resource, but I can if that's helpful in answering this question.

I talked to Dylan about this, and he reminded me that our logic for checking codes in value sets only looks in the `expansion` or the `compose` so there's nothing we should update in our ValueSets to include both, since both won't be checked.

**Review by jafeltra — COMMENTED** [review]
One last thought after your most recent changes: should we remove all the radiation procedure related value sets since we removed the extractor entirely?

**Pull request merged**